quarkchain/cluster/root_state.py

```diff
@@ -456,6 +456,10 @@ def validate_block(self, block, adjusted_diff: int = None):
                 block.header.height,
             )
             actual_coinbase_amount = block.header.coinbase_amount_map.balance_map
+
+            if self.env.cluster_config.GRPC_SLAVE_LIST:
```

Comment:
for this check and the one at line 493, I suggest adding two flags at the method parameters `skip_coinbase_check` and `skip_root_block_linkage`

Reply:
ok, then libra should not include prev_root_block_id in Minorblockheader